ocaml/sdk-gen/powershell/autogen/XenServerPSModule.psd1

```diff
@@ -41,29 +41,27 @@ CompanyName = 'Citrix Systems, Inc'
 Copyright = 'Copyright (c) Citrix Systems, Inc. All rights reserved.'
 
 # Requirements
-PowerShellVersion = '4.0'
+PowerShellVersion = '7.2'
 PowerShellHostName = ''
 PowerShellHostVersion = ''
-DotNetFrameworkVersion = '4.5'
-CLRVersion = '4.0.30319'
+DotNetFrameworkVersion = '6.0'
 ProcessorArchitecture = 'None'
 
 #Contents
 ModuleToProcess = 'XenServerPowerShell.dll'
 RequiredModules = @()
 NestedModules = @()
 ModuleList = @('XenServerPowerShell.dll')
-RequiredAssemblies = @('CookComputing.XmlRpcV2.dll',
-                       'Newtonsoft.Json.CH.dll',
+RequiredAssemblies = @('System.Management.Automation.dll',

```

Comment:
I presume this is needed for the Linux case? (Previously it was not necessary for Windows).